### PR TITLE
Mention Apple silicon support in release notes

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

**Comment by lptr** [comment]
@bot-gradle test QFL please.

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -62,6 +62,15 @@ File system watching and configuration caching is enabled for the comparison.
...
 You can find the performance test project [here](https://github.com/gradle/santa-tracker-performance).
 
+### Native support for Apple silicon
+
+With previous Gradle versions there were two options to run Gradle builds on the new Macs equipped with the M1 chip:
+
+* run with a native ARM JDK, but lose support for the rich console and file system watching, or
+* use an Intel JDK with Rosetta, but run at about half the performance.
+
+With Gradle 7.0 these limitations are gone, and Gradle now fully supports Apple silicon using a native ARM JDK.
```
Comment (melix) [inline subprojects/docs/src/docs/release/notes.md:72]: Well, actually maybe we should warn if you run with an Intel JDK? The sentence sounds like if you're using 7.0 then it doesn't matter if you use the Intel or ARM version, which is not true.

Comment (lptr) [reply subprojects/docs/src/docs/release/notes.md:72]: It does say "using a native ARM JDK" in the very sentence. Could you provide a concrete suggestion to improve this?

Comment (melix) [reply subprojects/docs/src/docs/release/notes.md:72]: It's the first part of the sentence which is confusing as it sound like the performance problem would be gone. What about:

> With Gradle 7 we added Apple Silicon support which means you should only use the native ARM JDK. You will get the performance, rich console and file system watching all together.

Comment (lptr) [reply subprojects/docs/src/docs/release/notes.md:72]: This is not correct. It is okay to use the Intel JDK if you want (you will need it if you use any embedded tool that has Intel-only native code in it).

My experience is that we also avoid using "we" and "you" in the release notes, though I'd be happy to change this if that's not correct.

Comment (melix) [reply subprojects/docs/src/docs/release/notes.md:72]: Then I don't have a better suggestion.

**Review by melix — APPROVED** [review]

**Pull request merged**